ansibledriver/service/ansible.py

```diff
@@ -539,6 +541,39 @@ def get_result(self):
       else:
         return LifecycleExecution(self.request_id, STATUS_COMPLETE, None, self.properties, self.associated_topology)
 
+    def _generate_additional_logs(self, result):
+      # Added logic to print logs for custom ansible module : ibm_cp4na_log_message
+      try:
+          if('message_direction' in result._result and 'external_request_id' in result._result and 'message_type' in result._result and 'protocol' in result._result):
+              message_direction = result._result['message_direction']
+              external_request_id = result._result['external_request_id']
+              content_type = result._result['content_type']
+              message_data = result._result['message_data']
+              message_type = result._result['message_type']
+              protocol = result._result['protocol']
+              protocol_metadata = result._result['protocol_metadata']
+
+              logging_context_dict = {'messageDirection' : message_direction, 'tracectx.externalRequestId' : external_request_id, 'ContentType' : content_type,
```

Comment:
Sorry guys, just noticed the format of these is wrong. It should be:
- `message_direction` instead of `messageDirection`
- `tracectx.externalrequestid` instead of `tracectx.externalRequestId`
- `content_type` instead of `ContentType`
- `message_type` instead of `messageType`

Reply:
We have just now changed the string values as you have suggested @dvaccarosenna
Code changes have been pushed.